app/javascript/components/App.js

```diff
@@ -17,10 +17,30 @@ import {
 } from "react-router-dom"
 
 class App extends Component {
+  constructor(props){
+    super(props)
+      this.state = {
+        collections: [] 
+      }
+    
+  }
+
+  readCollections = async () =>{ 
```

Comment:
Is everything rendering on the page properly?

Reply:
We were able to log the changes in state collections property. We have not worked on the display yet.